src/Adapter/AdapterInterface.php

```diff
@@ -36,3 +36,10 @@ public function getNormalizedIdentifier($model);
      */
     public function getUrlsafeIdentifier($model);
 }
+
+if (!interface_exists(\Sonata\CoreBundle\Model\Adapter\AdapterInterface::class)) {
```

Comment:
Can you please try replacing all this with simply `interface_exists(\Sonata\CoreBundle\Model\Adapter\AdapterInterface::class)` (no if, no `class_alias`). It should be sufficient to trigger the autoload (and that means that currently, the `class_alias` is never used :P